fedora/product.yml

```diff
@@ -10,13 +10,20 @@ pkg_manager: "dnf"
 
 init_system: "systemd"
 
-# The fingerprints below are retrieved from https://getfedora.org/keys/
-latest_version: 29
-latest_release_fingerprint: "5A03B4DD8254ECA02FDA1637A20AA56B429476B4"
-latest_pkg_release: "5a886537"
-latest_pkg_version: "429476b4"
+# The fingerprint and pkg_version are retrieved from https://getfedora.org/keys/
+future_version: 31
+future_release_fingerprint: "7D22D5867F2A4236474BF7B850CB390B3C3359C4"
+future_pkg_version: "3c3359c4"
```

Comment:
@matejak What is the intended use of future key?
The OVAL check doesn't use them.

Reply:
I have just filled that, as the information is available on the website.

Reply:
@matejak Should we just have it there unused?

I think it should be used in check and remediations (If we would like to test that this rule works in future fedora 31), or have it removed.

Reply:
What do you mean? If I just remove that, then anybody updating this rule for Fedora 31 five months from now will have to pull exactly that data from the website. Having it in the file will save time, and it doesn't break anything.

Reply:
I'm not strongly opposed to this, I just don't see much use in storing data for future use.

> Having it in the file will save time, and it doesn't break anything.

The time spent will be the same for every Fedora release, one will be filling in key data for `future_version` or `current_version`.

Reply:
Actually one can use `future_version` to populate `current_version` every other release, therefore not having to look at the website :-)

Reply:
Yes, for `current_version` it will be faster, but `future_version` will have to be populated again.
One may even run into the issue that `future_version` key might not exist yet, not sure what are the odds of this.